# error-discovery
调用自己创建的类时报错

运行demo文件夹下what.py报错，原因是？
